### ISSUE TITLE
Reworking textEquiv

### ISSUE THREAD

**Issue by MarcSaurette** [kateBrasseur/M.A.Thesis#3]
I am wondering whether the [Textequiv] should have abbreviations unpacked? If the idea here is accessibility, surely the full Latin words is preferable? Both scholars, students, and those with reading difficulties (or using machine reading) would appreciate a diplomatic edition (rather than just a transcription). 

Retaining a transcription = best for training Transcribus, helping people read what is on the page
Unpacking the abbreviations (diplomatic edition) = best for comprehensibility

For terminological clarity: a transcription ≠ an edition
